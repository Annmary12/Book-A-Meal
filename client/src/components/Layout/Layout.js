import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import NavBar from './NavBar/NavBarContainer.js';
import Footer from './Footer/Footer.js';
import styles from './layout.scss';

const Layout = ({ children }) => (
  <Fragment>
    <div className={styles.layout}>
      <NavBar />
      {children}
    </div>
    <Footer />
  </Fragment>
);

Layout.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired
};


export default Layout;
