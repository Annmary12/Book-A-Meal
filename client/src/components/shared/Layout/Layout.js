import React, { Fragment } from 'react';
import NavBar from './NavBar/NavBar.js';
import Footer from './Footer/Footer.js';

const Layout = props => (
  <Fragment>
    <NavBar />
    {props.children}
    <Footer />
  </Fragment>
);

Layout.propTypes = {
  children: React.PropTypes.oneOfType([
    React.PropTypes.arrayOf(React.PropTypes.node),
    React.PropTypes.node
  ]).isRequired
};

export default Layout;
