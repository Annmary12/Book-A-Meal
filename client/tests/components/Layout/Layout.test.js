import React from 'react';
import Layout from '../../../src/components/Layout/Layout.js';

describe('Layout component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Layout />);

    expect(wrapper).toMatchSnapshot();
  })
})