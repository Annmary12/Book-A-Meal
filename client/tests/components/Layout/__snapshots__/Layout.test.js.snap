// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should match snapshot 1`] = `
<Fragment>
  <div
    className="layout"
  >
    <Connect(NavBar) />
  </div>
  <Footer />
</Fragment>
`;
